checkfiles.py

```diff
@@ -109,6 +109,19 @@ def check_format(encValData, job, path):
     else:
         chromInfo = '-chromInfo={}/{}/chrom.sizes'.format(encValData, assembly)
 
+    # separate bam validation using samtools quickcheck
+    if item.get('file_format') == 'bam':
+        try:
+            output = subprocess.check_output(
+                ['samtools'] + ['quickcheck'] + [path], stderr=subprocess.STDOUT)
```

Comment:
Kinda nitpicking here, because that works as well. I think we could make this a bit nicer (and save some brackets) by either writing the command in one list `['samtools', 'quickcheck', path]` or by using `shlex` module (that is in the standard library and obviously needs to be imported) and write `shlex.split(f'samtools quickcheck {path}')`. I think I slightly prefer the former one because the command is quite simple.